### ISSUE TITLE
Stack_Overflow_* notebooks

### ISSUE THREAD

**Issue by connected-wwen** [connected-bsamadi/colab-notebooks#1]
Hey Behzad, I have a few comments on the `Stack_Overflow_Views_Neural_Network`, `Stack_Overflow_Reputation_Neural_Network`, and `Stack_Overflow_Reputation_Linear_Regression` notebooks. Let me know what you think. Thanks!

* Some features, such as `AVG(author.time_sec) AS avg_time`, does not constitute meaningful features and we should consider eliminating them (if validation loss is improved).
* Feature selection and engineering is often an iterative process, please consider working more on those areas and see if the results can be improved.
* We can see that in your neural network model training process, the validation loss is lower than the training loss by a large margin. There are many reasons that can happen but usually it means something is not right. Please investigate and try to explain why it happened in this case.
